### ISSUE TITLE
Dependencies suggestions nil pointer dereference

### ISSUE THREAD

**Issue by piotrkowalczuk** [Masterminds/glide#462]
`glide init` produce such output on OS X.

```
[INFO]	Looking for dependencies to make suggestions on
[INFO]	--> Scanning for dependencies not using verion ranges
[INFO]	--> Scanning for dependencies using commit ids
[INFO]	Gathering information on each dependency
[INFO]	--> This may take a moment. Especially on a codebase with many dependencies
[INFO]	--> Gathering release information for dependencies
[INFO]	--> Looking for dependency imports where versions are commit ids
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xb0 pc=0x667a2]

goroutine 1 [running]:
panic(0x3f2e20, 0xc420012080)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/Masterminds/glide/action.wizardFindVersions(0xc4202652c0)
	/Users/piotrkowalczuktravelaudience/Projects/go/src/github.com/Masterminds/glide/action/config_wizard.go:286 +0x792
github.com/Masterminds/glide/action.ConfigWizard(0x45305d, 0x1)
	/Users/piotrkowalczuktravelaudience/Projects/go/src/github.com/Masterminds/glide/action/config_wizard.go:60 +0x3f6
github.com/Masterminds/glide/action.Create(0x45305d, 0x1, 0x0)
	/Users/piotrkowalczuktravelaudience/Projects/go/src/github.com/Masterminds/glide/action/create.go:48 +0x33b
main.commands.func1(0xc4201b4000)
	/Users/piotrkowalczuktravelaudience/Projects/go/src/github.com/Masterminds/glide/glide.go:159 +0x9b
github.com/Masterminds/glide/vendor/github.com/codegangsta/cli.Command.Run(0x454653, 0x6, 0x453b1c, 0x4, 0x0, 0x0, 0x0, 0x467d93, 0x34, 0x0, ...)
	/Users/piotrkowalczuktravelaudience/Projects/go/src/github.com/Masterminds/glide/vendor/github.com/codegangsta/cli/command.go:179 +0xbba
github.com/Masterminds/glide/vendor/github.com/codegangsta/cli.(*App).Run(0xc420089080, 0xc42000c240, 0x2, 0x2, 0x0, 0x0)
	/Users/piotrkowalczuktravelaudience/Projects/go/src/github.com/Masterminds/glide/vendor/github.com/codegangsta/cli/app.go:196 +0xbfd
main.main()
	/Users/piotrkowalczuktravelaudience/Projects/go/src/github.com/Masterminds/glide/glide.go:122 +0x58d
```

**Comment by mattfarina** [comment]
@piotrkowalczuk thanks for filing the bug. Can you share the `glide.yaml` file it saved before creating this error? I'd like to track this problem down to fix it and that would help.

**Comment by mattfarina** [comment]
Found the bug. Will fix on Tuesday.

**Comment by piotrkowalczuk** [comment]
Thanks for fast response. Im not allowed to provide you `.yaml` file.

**Comment by mattfarina** [comment]
I believe this is fixed with the merger of #463. Please let me know if the issue is still happening.
